integration-cli/docker_cli_rm_test.go

```diff
@@ -7,6 +7,19 @@ import (
 	"github.com/go-check/check"
 )
 
+func (s *DockerSuite) TestRmContainerWithBtrfsSubvolume(c *check.C) {
+	testRequires(c, BtrfsSystem)
+
+	out, _, err := dockerCmdWithError("run", "--name", "testwithbtrfs", "fedora:latest", "/bin/sh", "-c", "apt-get -y -q install btrfs-progs && btrfs subvolume create /test")
```

Comment:
this test should not call fedora, please use an image we already have like busybox